{{ cookiecutter.repo_name }}/README.md

```diff
@@ -62,11 +62,29 @@
 
 ## Get Started
 
-1. Install project locally (use venv): `pip install -e {{cookiecutter.repo_name}}`
-1. Build project container: `docker build -t {{cookiecutter.repo_name}}:latest .`
-1. Start up local services: `cd {{cookiecutter.repo_name}}; docker-compose up -d`
-1. Run the example task: `luigi --module {{cookiecutter.repo_name}}.task Example`
+1. Set MODULES_PATH this is the directory that contains the current project as well as dependencies like sparsity and drtools `MODULES_PATH=..`
```

Comment:
Please point out that it must be an absolute path. The example you gave suggests that it may be relative.